modules/nf-core/tidk/plot/tests/main.nf.test

```diff
@@ -0,0 +1,73 @@
+nextflow_process {
+
+    name "Test Process TIDK_PLOT"
+    script "../main.nf"
+    process "TIDK_PLOT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "tidk"
+    tag "tidk/plot"
+
```

Comment:
```suggestion
tag "tidk/search"

```

Fixes linting error: `Tags do not adhere to guidelines. Tags missing in main.nf.test: tidk/search`